src/components/_modals/DepositModal/SommelierTab.tsx

```
@@ -402,135 +346,53 @@ export const SommelierTab = ({
-      { account: address }
-    )
+    const walletCapabilities = await getCapabilities(publicClient)
+    const atomicStatus =
+      walletCapabilities[cellarConfig.chain.wagmiId]?.atomic?.status
+
+    const canDoBatchCall =
+      atomicStatus === "ready" || atomicStatus === "supported"
+
+    const allowance = nativeDeposit
+      ? Number.MAX_SAFE_INTEGER
+      : // @ts-ignore
```

ESLint

[error] Do not use "@ts-ignore" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)